Sources/Clock.swift

```diff
@@ -1,5 +1,8 @@
 import Foundation
 
+/// Struct that has time + related metadata
+public typealias FullTime = (Date, TimeInterval)
```

Comment:
please use named arguments, i.e.

```
public typealias FullTime = (date: Date, timeSinceLastNtpSync: TimeInterval)
```

Reply:
added named arguments, and changed name to "AnnotatedTime"